### ISSUE TITLE
Can't see right icons in Linux Mint 17 Cinnamon

### ISSUE THREAD

**Issue by jpadron** [jonls/redshift#170]
Hi, I'm using redshift in Ubuntu 14.04 and everything is working fine, but in Linux Mint 17 (Cinnamon) I can't see the proper icons (only the red included in image above). I've compiled the master src without any problem.

Thanks in advance.

![seleccion_001](https://cloud.githubusercontent.com/assets/30490/5966356/98cd61ea-a7fb-11e4-993e-8779b67b9cfd.png)

**Comment by jonls** [comment]
So you are compiling redshift yourself? Are you installing redshift using `make install`?

**Comment by jpadron** [comment]
Yes, I've followed the same procedure as in Ubuntu. The app is working fine in Linux Mint, but something happens with the icons. They are not loaded correctly.

**Comment by gordon-morehouse** [comment]
I can confirm this and suspect these as possibly related:

https://bugs.launchpad.net/ubuntu/+source/redshift/+bug/677783
https://bugs.launchpad.net/ubuntu/+source/redshift/+bug/1188961

If you control the launchpad, can you migrate or close all the old bugs from there? If not, who can?
